recipe/meta.yaml

```diff
@@ -81,7 +81,7 @@ outputs:
         - libcurl 7.62.0
         - expat
         - freexl
-        - geos 3.7.0
```

Comment:
I just merged https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/155 and I'll restart rebuild gdal's stack from the beginning, it is the only way to avoid chasing our own tails here.